### ISSUE TITLE
Where should I put my few shot dataset?

### ISSUE THREAD

**Issue by winnie9802** [open-mmlab/mmfewshot#122]
Since few shot dataset is just for finetuning the model and the test.py won't save the change of the model, where should I put my fewshot dataset? training set or validation set? In that way, I could use the pth file to predict my images in the demo.py?
